### PR TITLE
chore: change travis clone depth

### PR DESCRIPTION
**Pull request by flx92**

### CODE REVIEW

**Comment by marionebl** [comment]
@flx92: Please explain what you are trying to achieve in the PR description. I added a commit that reintroduces `after_script`.

File: .travis.yml [hunk 1]
```diff
@@ -11,8 +11,9 @@ notifications:
 before_install:
   - npm install @alrra/travis-scripts
   - npm run auth
+install: git fetch --unshallow --tags
 script:
   - npm run build:ci
-after_success:
+after_sucess:
```
Comment (marionebl) [inline .travis.yml]: Readded this, let's tackle debuggability in another pr

Comment (timbru31) [inline .travis.yml]: typo (**sucess**)

**Comment by marionebl** [comment]
@timbru31 Please re-review

**Pull request merged**